src/core/ad.jl

```diff
@@ -108,38 +110,20 @@ gradient2(_vi::VarInfo, model::Function, spl::Union{Nothing, Sampler}) = begin
   g(vi[spl])
 end
 
-@init @require ReverseDiff="37e2e3b7-166d-5795-8a7a-e32c996b4267" begin
-
 gradient_r(theta::Vector{Float64}, vi::VarInfo, model::Function) = gradient_r(theta, vi, model, nothing)
 gradient_r(theta::Vector{Float64}, vi::Turing.VarInfo, model::Function, spl::Union{Nothing, Sampler}) = begin
```

Comment:
This line and the one preceding it overruns the 92 character limit. Probably an idea to refactor.
Also, do we really need the concrete `Vector{Float64}`? Would `AbstractVector{<:Real}` not suffice?

Reply:
(I appreciate you've not modified this line, but since we're here...)

Reply:
Also, documentation.